docs/source/transaction_family_specifications/integerkey_transaction_family.rst

```diff
@@ -0,0 +1,134 @@
+***************************************************
+IntegerKey Transaction Family Specification
+***************************************************
+
+Overview
+=========
+The IntegerKey transaction family allows users to set, increment, and decrement
+the value of entries stored in state. 
+
+An IntegerKey family transaction request is defined by the following values:
+
+* A verb which describes what action the transaction takes 
+* A name of the entry that is to be set or changed
+* A value by which the entry will be set or changed
+
+The 'set' verb is used to create new entries. The initial value of the entry will be 
+set to the value specified in the transaction request. The 'inc' and 'dec' verbs are 
+used to change the value of existing entries in state.
+
+State
+=====
+This section describes in detail how IntegerKey transaction information is stored 
+and addressed.
+
+The address values of IntegerKey transaction family entries are stored in state as
+a cbor encoded dictionary, with the key being the *Name* and the value being an integer 
+*Value*.
+
+.. code-block:: python
+
+    cbor({    
```

Comment:
So the cbor specification is making more sense to me. The first type of map is described this way: ```0xa0..0xb7      | map (0x00..0x17 pairs of data items follow) ```. So that is the first hex representation of the initial bytes for a map that has 23 or less key-value pairs in a map. So when the maps have 23 or less pairs, this would specify the format of the bytes stored in state. The other map specifications in cbor have to do with more pairs of items, and then also an indefinite amount.